modules/src/dice.py

```diff
@@ -3,14 +3,12 @@
 from templates.text import TextTemplate
 from templates.quick_replies import add_quick_reply
 
-dice_sides = {
-	1: u'\u2680',
-	2: u'\u2681',
-	3: u'\u2682',
-	4: u'\u2683',
-	5: u'\u2684',
-	6: u'\u2685'
-}
+dice_sides = {1: 'https://upload.wikimedia.org/wikipedia/commons/6/62/Kismet-Ace.png',
+              2: 'https://upload.wikimedia.org/wikipedia/commons/6/6d/Kismet-Deuce.png',
+              3: 'https://upload.wikimedia.org/wikipedia/commons/f/f2/Kismet-Trey.png',
+              4: 'https://upload.wikimedia.org/wikipedia/commons/1/11/Kismet-Four.png',
+              5: 'https://upload.wikimedia.org/wikipedia/commons/8/8e/Kismet-Five.png',
+              6: 'https://upload.wikimedia.org/wikipedia/commons/4/4e/Kismet-Six.png'}
 
 def process(input, entities=None):
     roll = TextTemplate(dice_sides[random.randint(1, 6)]).get_message()
```

Comment:
You'd have to use AttachmentTemplate to send images in the response. See https://github.com/swapagarwal/JARVIS-on-Messenger/blob/master/modules/src/coin.py for reference.